dvc/dvcfile.py

```diff
@@ -137,16 +136,17 @@ def merge(self, ancestor, other):
 
 class SingleStageFile(FileMixin):
     from dvc.schema import COMPILED_SINGLE_STAGE_SCHEMA as SCHEMA
+    from dvc.stage.loader import SingleStageLoader as LOADER
```

Comment:
Same here `stage` import is slow, making it a bit lazy.

Reply:
Seems a bit weird, maybe let's just import in place?

Reply:
We do it in the `Repo`. 🙂

Reply:
Touche :slightly_smiling_face: